### DISCUSSION TITLE
Cannot create log file /var/log/motion/motion.log: Permission denied

### DISCUSSION THREAD

**Discussion by yaomtc** [Motion-Project/motion#1699] [Q&A]
### Did you read the guide?

Yes

### What is the base version number of Motion being used?

4.3.x

### What was the install method?

Installed via package tool

### What is base architecture?

ARM-32bit

### What is the distro being used?

Raspbian

### Disto version number

11 (bullseye)

### Camera type(s) being used?

v4l2 USB camera

### Describe the issue/problem and steps to reproduce

It won't start due to a permissions issue.

### Motion log output at log_level 8

```shell
[0:motion] [NTC] [ALL] conf_load: Processing thread 0 - config file /etc/motion/motion.conf
[0:motion] [ERR] [ALL] myfopen: Error opening file /var/log/motion/motion.log with mode a: Permission denied
[0:motion] [EMG] [ALL] motion_startup: Exit motion, cannot create log file /var/log/motion/motion.log: Permission denied

Broadcast message from systemd-journald@rpi (Thu 2023-08-03 14:33:52 CDT):

motion[5427]: [0:motion] [EMG] [ALL] motion_startup: Exit motion, cannot create log file /var/log/motion/motion.log: Permission denied


Message from syslogd@rpi at Aug  3 14:33:52 ...
 motion[5427]: [0:motion] [EMG] [ALL] motion_startup: Exit motion, cannot create log file /var/log/motion/motion.log: Permission denied
```

**Comment by olddog55** [comment]
systemd tends to screw up simple things

Does /var/log/motion exist?
What are the read/write permissions on /var/log/motion?
Is there a conflicting /var/log/motion/motion.log? ( 'ls -la /var/log/motion/' can provide this )

You may need to do:
```  sudo mkdir /var/log/motion ```
```  sudo chown motion /var/log/motion ```

And verify that the /etc/logrotate.d/motion file contains your appropriate log rotation config.
